invenio_app_ils/mappings/v6/documents/document-v1.0.0.json

```diff
@@ -39,6 +39,14 @@
             }
           }
         },
+        "_access": {
```

Comment:
the read should be an array of keywords

Reply:
I think it's ok to leave it like this, as ES handles array types transparently